lib/modules/manager/bitbucket-pipelines/extract.ts

```diff
@@ -59,3 +39,43 @@ export function extractPackageFile(content: string): PackageFileContent | null {
   }
   return { deps };
 }
+function addDepAsBitbucketTag(
+  deps: PackageDependency<Record<string, any>>[],
+  pipe: string
+): void {
+  const [depName, currentValue] = pipe.split(':');
+
+  const dep: PackageDependency = {
+    depName,
+    currentValue,
+    datasource: BitBucketTagsDatasource.id,
+  };
+
+  logger.trace(
+    {
+      depName: dep.depName,
+      currentValue: dep.currentValue,
+    },
+    'Bitbucket pipe tag'
+  );
```

Comment:
i think we can remove those trace, as we get that data logged as debug later inside `package files with updates`

@rarkins WDYT?

Reply:
Sure. We're quite inconsistent with trace logging today. In this case it doesn't seem to be adding anything new

Reply:
Trace logging has been removed